### ISSUE TITLE
[FEATURE REQUEST] Design API collapses multiple bearer tokens into one variable reference for all debug requests

### ISSUE THREAD

**Issue by gh-andre** [getinsomnia/insomnia#4271]
### Expected Behavior

When multiple bearer token authentication schemes are specified in the API design document, debug requests that are auto-inserted on the Debug tab should use different variables to reference each type of bearer token.

### Actual Behavior

Debug requests auto-inserted on the Debug tab all use the same variable name - `bearerToken`, which requires manual edits for all of them every time the design document is changed.

### Reproduction Steps

* Create a new design document with multiple bearer token schemes:

```yaml
openapi: "3.0.0"
info:
 title: 'ABC'
 version: '0.1.0'
components:
  securitySchemes:
    bearerA:
      type: 'http'
      scheme: 'bearer'
    bearerB:
      type: 'http'
      scheme: 'bearer'
paths:
  /abc/:
    get:
      summary: 'Uses bearerA'
      security:
      - bearerA: []
      responses:
       '200':
         description: 'Success'
  /xyz/:
    get:
      summary: 'Uses bearerB'
      security:
      - bearerB: []
      responses:
       '200':
         description: 'Success'
```

* Click Authorize button in the preview pane of the design document - there will be authorization input box for each bearer token.
* Switch to the Debug tab and wait for both requests to appear in the list.
* Switch to the Bearer tab and click each request - they all reference `bearerToken` variable, even though their authentication schemes are different in the design document, so each must be edited manually before it can be used.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

One workaround I found is to use folder environment to give different values to the same variable, such as `bearerToken`. Not sure if this is the intended behavior for same-name variables on the Debug tab - I cannot find any docs on this.

It's usable, but less descriptive because type of bearer token isn't immediately visible in the request and the environment need to be inspected to see the mapping of `bearerToken` to the actual token defined, for example, in the parent environment.

### Insomnia Version

2021.6.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Pro Version	21H2 OS build	19044.1387

### Installation method

Insomnia website download

**Comment by filfreire** [comment]
Hi @gh-andre, thank you for reporting this issue!

From what've seen looking at the codebase, we're hardcoding the `bearerToken` variable (and its value as `bearerToken` always when we detect the authentication scheme is Bearer:
<img width="376" alt="Screen Shot 2022-05-10 at 12 53 49" src="https://user-images.githubusercontent.com/11976836/167622593-f962faa1-7371-4033-ad28-e24272df26f1.png">

This explains why the Authorization set in the Spec preview:
<img width="650" alt="Screen Shot 2022-05-10 at 12 51 47" src="https://user-images.githubusercontent.com/11976836/167622891-19f62884-0dd0-49c8-91cb-9a98c9d1b7e0.png">

Has no effect on the actual Debug generated requests, and you see what you reported:
<img width="532" alt="Screen Shot 2022-05-10 at 12 51 39" src="https://user-images.githubusercontent.com/11976836/167622957-b6d901b7-2d1e-463f-b809-4f3584850f46.png">


In summary: It seems the functionality you describe is not supported, and there's at the moment no connection between the `Authorize` button, and filling in that data, and what is generated on the `Debug` tab. Nevertheless, it is an possibly interesting improvement we could pursue.

Tagging @wdawson and marking this as an improvement. We will update this ticket if there's any new developments. In the meantime @gh-andre feel free to report any other issues you find 👍
